src/components/HomePage/components/Hero/styles.js

```diff
@@ -0,0 +1,57 @@
+import styled from 'styled-components';
+
+import { colours } from '../../../../colours';
+
+export const Background = styled.div`
+    background-color: ${colours.black};
+    padding: 1rem 0;
+`;
+
+export const Wrapper = styled.div`
+    align-items: center;
+    display: flex;
+    justify-content: space-between;
+    margin: 10px auto;
+    width: 950px;
+`;
+
+export const IntroWrapper = styled.div`
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    width: 450px;
+`;
+
+export const Subtitle = styled.h3`
+    color: ${colours.primaryColour};
+    font-family: 'Quicksand', sans-serif;
+    font-size: 36px;
+    margin: 0 0 2rem 0;
+    text-align: center;
+`;
+
+export const ButtonCTA = styled.button.attrs({
+    type: 'button'
+})`
+    background: linear-gradient(180deg, #20fff2 0%, rgba(255, 255, 255, 0) 100%), #00a7cb;
```

Comment:
can we create colour tokens for these? also perhaps look into using https://polished.js.org for utils?